pkg/controller/volume/persistentvolume/deprecated-recycle-api.md

```diff
@@ -0,0 +1,32 @@
+## Deprecation of Persistent Volume Recycler Behavior
```

Comment:
I don't know where this file should live, but the code is probably not the right place.  Maybe in the API docs repo, linked from the docs about the feature?

@devin-donnelly